Added boolen to refresh a cache image.

chetan-cueclad opened the pull request:
A scenario where an image is updated or replaced by the user e.g. Profile Pic - Currently there is no mechanism to tell the cache to refresh image.
This PR adds a parameter where the cache of an image is updated as soon as the pic is updated/replaced.
```
Image(
  image: PCacheImage(
    'gs://flutter-template-44674.appspot.com/images/user/0ooAw4dX5AeGhkH1JYkoWcdwvc72_big.jpg',
    clearCacheImage: true
  )
)

chetan-cueclad committed:
One parameter added in getImage method.

Actually we clear single image.

chetan-cueclad committed:
One parameter added to getImage method.

chetan-cueclad committed:
One parameter added to getImage method.

chetan-cueclad committed:
Update paulonia_cache_image_web.dart

ChrisChV reviewed (changes requested):
Hi @chetan-cueclad Thanks for the contribution!
It's a great feature that was missing. Can you add this same functionality to the in-memory cache on `InMemoryManager`

chetan-cueclad committed:
New functionality added.
1. Clear memory image.
2. Clear entire cache images from storage.

chetan-cueclad committed:
New functionality added.
1. Clear memory image.
2. Clear entire cache images from storage.

chetan-cueclad committed:
Rename clearImg to clearCacheImage.

kaumudpa committed:
Added a method and a parameter

Updated readme to include `clearAllCacheImages()` method and `clearCacheImage` parameter

kaumudpa commented:
Added another method `clearAllCacheImages();` to clear the entire cache and Updated Readme to include `clearCacheImage`. and changes done as per your request.

ChrisChV commented:
Thanks for the contribution! @kaumudpa the `clearAllCacheImages` is a great functionality. I have thought of creating a small module so that from any part of the code you can delete or add images to the cache: #5 
The function `clearAllCacheImages` can be in that module perfectly
